plugins/modules/aci_endpoint_mac_tag.py

```diff
@@ -0,0 +1,311 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Sabari Jaganathan <sajagana@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_endpoint_mac_tag
+short_description: Manage Endpoint MAC Tag (fv:EpMacTag)
+description:
+- Manage Endpoint MAC Tag on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - The name of the Tenant.
+    type: str
+  name:
+    description:
+    - The name of the Endpoint MAC Tag.
+    type: str
+  endpoint_mac_address:
+    description:
+    - The MAC address of the Endpoint MAC Tag.
+    type: str
+    aliases: [ mac_addr ]
+  bd:
```

Comment:
is it required for bd to exist? if so add section in notes

Reply:
Done